Tasks/DownloadBuildArtifactsV0/package.json

```diff
@@ -19,6 +19,6 @@
   "dependencies": {
     "vsts-task-lib": "2.1.0",
     "vso-node-api": "^6.2.8-preview",
-    "artifact-engine": "0.1.25"
```

Comment:
update task version?

Reply:
already updated to 0.146.0 last time...that's not deployed yet